packages/core/scripts/umip-3/4_ExecuteSimulate.js

```diff
@@ -269,7 +269,7 @@ async function runExport() {
     expirationTimestamp: "1590969600", // one week contract
     collateralAddress: collateralToken.address,
     priceFeedIdentifier: web3.utils.utf8ToHex("ETHBTC"),
-    syntheticName: "Test UMA Token",
+    syntheticName: "Test Synthetic Token",
     syntheticSymbol: "ETHBTCSynth",
```

Comment:
Should this be `SYNTH`?

Reply:
Actually, I'm considering reverting this change completely because it's in a UMIP script. WDYT?